ui/src/message_popup/author/edit_question_page.jsx

```diff
@@ -0,0 +1,35 @@
+/* @flow weak */
+import React from 'react';
+import _ from 'lodash';
+
+import {allQuestions} from '../questions.js';
+import {withStudents, withLearningObjectiveAndIndicator} from '../transformations.jsx';
+
+import QuestionPage from './question_page.jsx';
+
+export default React.createClass({
+  displayName: 'EditQuestionPage',
+
+  propTypes: {
+    questionId: React.PropTypes.string.isRequired
+  },
+
+  render(){
+    const question = withLearningObjectiveAndIndicator(_.find(withStudents(allQuestions), question => question.id.toString()===this.props.questionId));
+    var goodExamplesText = '';
+    _.forEach(question.examples, example => {goodExamplesText += (goodExamplesText === '' ? '' : '\n\n') + example;});
```

Comment:
Prefer `const` over `var` wherever you can, since it provides a guarantee that the value won't be changed later accidentally.  In this case would this work?

```
const goodExamplesText = question.examples.join('\n');
```